docs/CHANGELOG.asciidoc

```diff
@@ -39,6 +40,395 @@ This section summarizes the changes in each release.
+Management::
+* Inspect action shows on dashboard for every chart {kibana-pull}65998[#65998]
+* Fixes an issue in Watcher, where a watch status or action status was incorrectly marked as "Error" {kibana-pull}67952[#67952]
+* Error handling {kibana-pull}68809[#68809]
+* Fixes an issue in Dev Tools Console where the example shown in the "Help" panel was formatted incorrectly {kibana-pull}71188[#71188]
+* Fixes the alignment of the Timing field for Warm, Cold and Delete phases on the Index Lifecycle Policy edit page {kibana-pull}71273[#71273]
+* Adopts data stream API changes {kibana-pull}71682[#71682]
+* Fixes #66185 {kibana-pull}66186[#66186]
+Maps::
+* Do not check count for blended layers when layer is not visible {kibana-pull}66460[#66460]
+* Fixes mapbox glyphs error when EMS access is turned off {kibana-pull}67427[#67427]
+* Fixes fit to bounds requests not getting canceled {kibana-pull}67629[#67629]
+* Fixes cannot select Solid fill-color when removing fields {kibana-pull}70621[#70621]
+* Fixes zoom in/zoom out buttons are not visible in dark mode {kibana-pull}72699[#72699]
+* Fixes removing global filter from layer can cause app to start thrashing {kibana-pull}72763[#72763]
+* Fixes cloned clustered documents layer returns error {kibana-pull}72975[#72975]
+* Fixes data driven style properties not working when cloned layer contains joins {kibana-pull}73124[#73124]
+* Fixes tile layer attibution text and attribution link validation errors {kibana-pull}73160[#73160]
+* Fixes fit to data for Point to Point layer {kibana-pull}73563[#73563]
+* Fixes #fit to bounds for ES document layers with joins {kibana-pull}73985[#73985]
+* Removes no longer required div wrapper around ValidatedDualRange {kibana-pull}70188[#70188]
+Metrics::
+* Fixes asynchronicity and error handling in Snapshot API {kibana-pull}70503[#70503]
+* Fixes evaluating rate-aggregated alerts when there's no normalized value {kibana-pull}73545[#73545]
+Monitoring::
+* Adds a case for Alerting if security/ssl is disabled {kibana-pull}71846[#71846]
+* Checks for security feature first when entering setup mode {kibana-pull}73821[#73821]
+Operations::
+* Manages pid folder {kibana-pull}68262[#68262]
+Platform::
+* Sets wrap to the errorLink so it doesn't go outside of box {kibana-pull}67797[#67797]
+* Fixes plugin lifecycle log to only include server plugins {kibana-pull}68686[#68686]
+Querying & Filtering::
+* When using KQL or the filter bar, if a request is issued to Elasticsearch for suggestions for a value for a specific field, the request 
+will be cancelled when navigating away as well as when the input is updated (and a new request is issued {kibana-pull}69769[#69769]
+Reporting::
+* Fixes a bug where the Kibana server could crash if the Reporting server-side headless browser crashes {kibana-pull}71989[#71989]
+Security::
+* Clones role button going to edit page {kibana-pull}70768[#70768]
```

Comment:
```suggestion
```

Sorry for the mis-label. This was a bug caught prior to shipping, and shouldn't be included as part of the release notes